include/pmacc/math/vector/Vector.hpp

```diff
@@ -513,6 +503,8 @@ namespace pmacc
             }
         };
 
+        static_assert(std::is_trivially_default_constructible_v<Vector<float, 3>>, "");
```

Comment:
Assuming you are still on C++14. Otherwise we can remove the `, ""`.

Reply:
Ok, `std::is_trivially_default_constructible_v` is C++17. I have not been around C++14 for years. Will fix it.

Reply:
offline discussed: this check should be part of the vector class and should use the type and dimension from the vector.

Reply:
I removed the check now because it got too complicated. I cannot put it inside the class, because at the time of the check, the class is still incomplete.